cbv/models.py

```diff
@@ -361,7 +366,7 @@ def basic_yuml_data(self, first=False):
         self._basic_yuml_data = yuml_data
         return self._basic_yuml_data
 
-    def basic_yuml_url(self):
+    def basic_yuml_url(self) -> str | None:
         data = ", ".join(self.basic_yuml_data(first=True))
         if not data:
             return None
```

Comment:
*Thought (out of scope):* It looks like this branch can't happen at the moment...